### PR TITLE
Collections and Data Structures

### PR DESCRIPTION
**Pull request by stevehoag**
Addressed comments and fixed <> formatting

### CODE REVIEW

File: docs/essentials/collections/Collections-and-Data-Structures.md [hunk 1]
```diff
@@ -40,26 +42,26 @@ In general, you should use generic collections. The following table describes so
...
+Use data Last-In-First-Out (LIFO) | [System.Collections.Generic.Stack&lt;T&gt;](http://dotnet.github.io/api/System.Collections.Generic.Stack%601.html) | [System.Collections.Stack](http://dotnet.github.io/api/System.Collections.Stack.html)
+Access items sequentially | [System.Collections.Generic.LinkedList&lt;T&gt;](http://dotnet.github.io/api/System.Collections.Generic.LinkedList%601.html) | No recommendation
+Receive notifications when items are removed or added to the collection. (implements [INotifyPropertyChanged](http://dotnet.github.io/api/System.ComponentModel.INotifyPropertyChanged.html) and [System.Collections.Specialized.INotifyCollectionChanged](http://dotnet.github.io/api/System.Collections.Specialized.INotifyCollectionChanged.html)) | [System.Collections.ObjectModel.ObservableCollection&lt;T&gt;](http://dotnet.github.io/api/System.Collections.ObjectModel.ObservableCollection%601.html) | No recommendation
+A sorted collection | [System.Collections.Generic.SortedList&lt;TKey, TValue&gt;](http://dotnet.github.io/api/System.Collections.Generic.SortedList%602.html) | [System.Collections.SortedList](http://dotnet.github.io/api/System.Collections.SortedList.html)
```
Comment (cartermp) [inline docs/essentials/collections/Collections-and-Data-Structures.md]: Should that be "Use a sorted collection"?

File: docs/essentials/collections/Collections-and-Data-Structures.md [hunk 2]
```diff
@@ -40,26 +42,26 @@ In general, you should use generic collections. The following table describes so
...
+A set for mathematical functions | [System.Collections.Generic.HashSet&lt;T&gt;](http://dotnet.github.io/api/System.Collections.Generic.HashSet%601.html), [System.Collections.Generic.SortedSet&lt;T&gt;](http://dotnet.github.io/api/System.Collections.Generic.SortedSet%601.html) | No recommendation
```
Comment (cartermp) [inline docs/essentials/collections/Collections-and-Data-Structures.md]: Should that be "Use a set for mathematical functions"?

File: docs/essentials/collections/Collections-and-Data-Structures.md (hunk 1 above)
Comment (stevehoag) [inline docs/essentials/collections/Collections-and-Data-Structures.md]: Probably should be, although it wasn't that way in the MSDN article. I'll fix it.

**Comment by stevehoag** [comment]
@cartermp  Fixed. Ready to merge?

**Comment by cartermp** [comment]
Not quite sure I like the affinity between `HashSet<T>` and performing mathematical functions.  What was the rationale for this article when it was written for MSDN?

**Comment by stevehoag** [comment]
@cartermp  Not sure who originally wrote this nor what their rationale was. Should I just remove this from the table?

**Comment by cartermp** [comment]
@stevehoag I think we can keep in the table.  The key benefit to a `HashSet<T>` for me has always been storing unique elements with similar performance characteristics of a `Dictionary<K, V>`.  Capturing that somehow (e.g., "Efficient storage and access of unique elements") seems like an appropriate value proposition.

**Comment by stevehoag** [comment]
@cartermp Good call - changed it to "Manage efficient storage and access of unique elements "

**Comment by cartermp** [comment]
LGTM.

**Pull request merged**